include/zrepl_mgmt.h

```diff
@@ -197,6 +197,8 @@ typedef struct zvol_info_s {
 	uint64_t 	write_byte;
 	uint64_t	sync_req_ack_cnt;
 	uint64_t 	sync_latency;
+	uint64_t 	cur_io_cnt;
```

Comment:
how about changing this to inflight_io_cnt?
Also, please add description for these two variables, for ex., 'dispatched_io_cnt' includes the 'inflight_io_cnt' as well.